app/src/main/java/com/stacker/simplegithubapp/MainActivity.kt

```diff
@@ -2,11 +2,35 @@ package com.stacker.simplegithubapp
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import com.stacker.simplegithubapp.databinding.ActivityMainBinding
+import kotlinx.android.synthetic.main.activity_main.*
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity(){
+
+    private lateinit var binding: ActivityMainBinding
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this,R.layout.activity_main)
         setContentView(R.layout.activity_main)
```

Comment:
```suggestion
        binding = DataBindingUtil.setContentView(this,R.layout.activity_main)
```
DataBindingUtil을 통해서 setContentView 즉, 뷰를 그리는걸 완료했다면, 또다시 setContentView를 할 필요가 없음.

```suggestion
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
```
사실 이런식으로도 할수이슴 ㅎ